uber/automated_emails.py

```diff
@@ -427,6 +427,14 @@ def __init__(self, subject, template, filter, ident, **kwargs):
     when=days_before(1, c.FINAL_EMAIL_DEADLINE),
     ident='volunteer_shift_schedule')
 
+if c.VOLUNTEER_AGREEMENT_ENABLED:
```

Comment:
This will completely remove the email from the system if `c.VOLUNTEER_AGREEMENT_ENABLED` is False.